keep-ui/app/workflows/[workflow_id]/executions.tsx

```diff
@@ -153,98 +147,178 @@ export default function WorkflowDetailPage({
+              <div>
+                <h1 className="text-2xl font-bold">
+                  {formatNumber(data.count ?? 0)}
+                </h1>
               </div>
-              <WorkflowGraph
-                showLastExecutionStatus={false}
-                workflow={workflow}
-                limit={executionPagination.limit}
-                showAll={true}
-                size="sm"
-              />
-              <h1 className="text-xl font-bold mt-4">Execution History</h1>
-              <TableFilters workflowId={data.workflow.id} />
-              <ExecutionTable
-                executions={data}
-                setPagination={setExecutionPagination}
-              />
-            </div>
-          )}
+            </StatsCard>
+            <StatsCard data={`${data.passCount}/${data.failCount}`}>
+              <Title>Pass / Fail ratio</Title>
+              <div>
+                <h1 className="text-2xl font-bold">
+                  {formatNumber(data.passCount)}
+                  {"/"}
+                  {formatNumber(data.failCount)}
+                </h1>
+              </div>
+            </StatsCard>
+            <StatsCard>
+              <Title>Success %</Title>
+              <div>
+                <h1 className="text-2xl font-bold">
+                  {(data.count
+                    ? (data.passCount / data.count) * 100
+                    : 0
+                  ).toFixed(2)}
+                  {"%"}
+                </h1>
+              </div>
+            </StatsCard>
+            <StatsCard>
+              <Title>Avg. duration</Title>
+              <div>
+                <h1 className="text-2xl font-bold">
+                  {(data.avgDuration ?? 0).toFixed(2)}
+                </h1>
+              </div>
+            </StatsCard>
+            <StatsCard>
+              <Title>Involved Services</Title>
+              <WorkflowSteps workflow={parsedWorkflowFile} />
+            </StatsCard>
+          </div>
+          <WorkflowGraph
+            showLastExecutionStatus={false}
+            workflow={workflow}
+            limit={executionPagination.limit}
+            showAll={true}
+            size="sm"
+          />
+          <h1 className="text-xl font-bold mt-4">Execution History</h1>
+          <TableFilters workflowId={data.workflow.id} />
+          <ExecutionTable
+            executions={data}
+            setPagination={setExecutionPagination}
+          />
         </div>
-      </Card>
-      {!!data.workflow && !!getTriggerModalProps && (
+      )}
+      {!!data?.workflow && !!getTriggerModalProps && (
         <AlertTriggerModal {...getTriggerModalProps()} />
       )}
     </>
   );
+};
+
+export default function WorkflowDetailPage({
+  params,
+}: {
+  params: { workflow_id: string };
+}) {
+  const router = useRouter();
+  const { data: session, status } = useSession();
+  const [navlink, setNavLink] = useState("overview");
+
+  const apiUrl = getApiURL();
```

Comment:
```suggestion
  const apiUrl = useApiUrl();
```
Now you need to use this client hook to get API url. Please import it.